### PR TITLE
feat(pie-docs): Add accessibility section

### PR DESCRIPTION
**Pull request by FayeCarter**
[Added] - Accessibility section to doc site
[Added] - Accessibility overview page
[Updated] - Nav and Footer components to show accessibility section

### CODE REVIEW

**Review by webbertakken — DISMISSED** [review]
Excellent work!

File: apps/pie-docs/src/accessibility/accessibility-overview.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    caption: "Inclusive design is better design for everyone, no user should be excluded because of disability."
+} %}
+
+---
+
+## Why is accessibility important?
+
+Just as a good recipe caters to the diverse tastes and dietary needs of all diners, a design system that incorporates accessibility ensures that all users, **regardless of their abilities or disabilities**, can access and use digital products and services.
+
+Incorporating accessibility into a design system can lead to a **better user experience, increased engagement**, and **increased customer loyalty**.
```
Comment (jamieomaguire) [inline apps/pie-docs/src/accessibility/accessibility-overview.md:26]: I might be wrong here but I think when using bold on words you're meant to apply it to **each**, **individual**, **word** and **not** the comma 😄

File: apps/pie-docs/src/accessibility/accessibility-overview.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+The build-it-once, use-it-everywhere model means the **accessibility knowledge** of the designers and developers who build these components is available to all of JET and our partners.
+
+Many accessibility features come free in the components, but it's important to make sure that components are integrated in a way that doesn't create **unforeseen accessibility barriers**. Depending on how components are used, there may be more design and implementation considerations.
+
+---
+
+## Assistive technology support
+
+Our components are tested for accessibility with automated and manual techniques. Users should expect to be able to access features built with our components using **modern assistive technologies**, including native and **third-party tools** like screen readers, speech recognition programs, supports for low vision and colour blindness, alternative keyboards, switch devices, and tools for readability.
```
Comment (jamieomaguire) [inline apps/pie-docs/src/accessibility/accessibility-overview.md:46]: Do we want to mention any of the particular programs and screenreaders we should test on?

File: apps/pie-docs/src/accessibility/accessibility-overview.md (hunk 1 above)
Comment (jordifornellsJET) [reply apps/pie-docs/src/accessibility/accessibility-overview.md:26]: Agreed, thanks for noticing this!

File: apps/pie-docs/src/accessibility/accessibility-overview.md (hunk 2 above)
Comment (jordifornellsJET) [reply apps/pie-docs/src/accessibility/accessibility-overview.md:46]: @jamieomaguire We have a 'Testing for accessibility' section in which we give more details. I can link to that page so we don't overwhelm people in this page already.

I would suggest having a look at that page yourself and adding/removing/changing any bits, if possible, thanks!

File: apps/pie-docs/src/accessibility/accessibility-overview.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+---
+
+## Why is accessibility important?
+
+Just as a good recipe caters to the diverse tastes and dietary needs of all diners, a design system that incorporates accessibility ensures that all users, **regardless of their abilities or disabilities**, can access and use digital products and services.
+
+Incorporating accessibility into a design system can lead to a **better user experience, increased engagement**, and **increased customer loyalty**.
+
+Finally, it’s important to highlight that designing with accessibility in mind can help organisations comply with accessibility standards and regulations, avoiding **legal and financial consequences**.
```
[the remark this answers is not in the document]
Comment (jordifornellsJET) [reply apps/pie-docs/src/accessibility/accessibility-overview.md:28]: Good point, let's make this change @FayeCarter

File: apps/pie-docs/src/accessibility/accessibility-overview.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
+---
+eleventyNavigation:
+    key: Accessibility overview
+    parent: Accessibility
+    order: 1
+title: Accessibility overview
+description: In this section you’ll find useful information about accessibility and why it is important at JustEatTakeaway.com
```
[the remark this answers is not in the document]
Comment (jordifornellsJET) [reply apps/pie-docs/src/accessibility/accessibility-overview.md:7]: Thanks for this, let's make this change @FayeCarter

File: apps/pie-docs/src/accessibility/accessibility-overview.md (hunk 2 above)
Comment (jordifornellsJET) [reply apps/pie-docs/src/accessibility/accessibility-overview.md:46]: @FayeCarter let's add a link to the Testing for accessibility page in this paragraph, I'll add it to designs

File: apps/pie-docs/src/accessibility/accessibility-overview.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+    src:"../../../assets/img/accessibility/accessibility-overview/accessibility-introduction.svg",
+    mobileSrc:"../../../assets/img/accessibility/accessibility-overview/accessibility-introduction_narrow.svg",
+    caption: "Inclusive design is better design for everyone, no user should be excluded because of disability."
+} %}
+
+---
+
+## Why is accessibility important?
+
+Just as a good recipe caters to the diverse tastes and dietary needs of all diners, a design system that incorporates accessibility ensures that all users, **regardless of their abilities or disabilities**, can access and use digital products and services.
```
[the remark this answers is not in the document]
Comment (jordifornellsJET) [reply apps/pie-docs/src/accessibility/accessibility-overview.md:24]: 👍 (**Done in design** ✅)

File: apps/pie-docs/src/accessibility/accessibility-overview.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (FayeCarter) [reply apps/pie-docs/src/accessibility/accessibility-overview.md:7]: 👍

File: apps/pie-docs/src/accessibility/accessibility-overview.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (FayeCarter) [reply apps/pie-docs/src/accessibility/accessibility-overview.md:24]: 👍

File: apps/pie-docs/src/accessibility/accessibility-overview.md (hunk 1 above)
Comment (FayeCarter) [reply apps/pie-docs/src/accessibility/accessibility-overview.md:26]: 👍

File: apps/pie-docs/src/accessibility/accessibility-overview.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (FayeCarter) [reply apps/pie-docs/src/accessibility/accessibility-overview.md:28]: 👍

File: apps/pie-docs/src/accessibility/accessibility-overview.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+## How we build inclusive experiences
+
+Using our components is a way to improve accessibility and consistency when building products for JustEatTakeaway.com.
+
+The component library in this documentation site includes code that we can use across applications, and this code includes **accessible markup**. Since the code exists in a single component that gets reused, it's easier to update and fix any bugs.
+
+The build-it-once, use-it-everywhere model means the **accessibility knowledge** of the designers and developers who build these components is available to all of JET and our partners.
+
+Many accessibility features come free in the components, but it's important to make sure that components are integrated in a way that doesn't create **unforeseen accessibility barriers**. Depending on how components are used, there may be more design and implementation considerations.
```
[the remark this answers is not in the document]
Comment (FayeCarter) [reply apps/pie-docs/src/accessibility/accessibility-overview.md:40]: 👍

File: apps/pie-docs/src/accessibility/accessibility-overview.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+Incorporating accessibility into a design system can lead to a **better user experience, increased engagement**, and **increased customer loyalty**.
+
+Finally, it’s important to highlight that designing with accessibility in mind can help organisations comply with accessibility standards and regulations, avoiding **legal and financial consequences**.
+
+---
+
+## How we build inclusive experiences
+
+Using our components is a way to improve accessibility and consistency when building products for JustEatTakeaway.com.
```
[the remark this answers is not in the document]
Comment (FayeCarter) [reply apps/pie-docs/src/accessibility/accessibility-overview.md:34]: 👍

**Pull request closed without merging**